pkg/apiserver/registry/networkpolicy/util.go

```diff
@@ -33,3 +39,43 @@ func GetSelectors(options *internalversion.ListOptions) (string, labels.Selector
 	key, _ := field.RequiresExactMatch("metadata.name")
 	return key, label, field
 }
+
+// PaginateMemberList returns paginated results if meaningful options are provided, options should never be nil.
+// Paginated results are continuous only when there is no member change across multiple calls.
+// Pagination is not processed if either page number or limit = 0, thus returns full member list.
```

Comment:
thus ... -> "in which the full member list is returned"?

Reply:
processed -> enabled?

Reply:
Updated.